x-pack/legacy/plugins/ml/public/explorer/explorer_charts/explorer_chart_distribution.js

```diff
@@ -407,72 +407,93 @@ export const ExplorerChartDistribution = injectI18n(class ExplorerChartDistribut
       // Show the time and metric values in the tooltip.
       // Uses date, value, upper, lower and anomalyScore (optional) marker properties.
       const formattedDate = formatHumanReadableDateTime(marker.date);
-      let contents = `${formattedDate}<br/><hr/>`;
+      const tooltipData = [{ name: formattedDate }];
+      const seriesKey = config.detectorLabel;
 
       if (_.has(marker, 'entity')) {
-        contents += `<div>${marker.entity}</div>`;
+        tooltipData.push({
```

Comment:
If it isn't easily possible to add the field name in here, I think this would look better left aligning the field value:

![image](https://user-images.githubusercontent.com/7405507/66818563-69de6b00-ef35-11e9-8f8c-e657a4e03882.png)